src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -175,41 +166,41 @@ public async Task Success()
                     Assert.False(publicationViewModel.IsSuperseded);
 
                     Assert.Equal(2, publicationViewModel.Releases.Count);
-                    Assert.Equal(expectedReleaseVersion1.Id, publicationViewModel.LatestReleaseId);
 
-                    Assert.Equal(expectedReleaseVersion1.Id, publicationViewModel.Releases[0].Id);
-                    Assert.Equal(expectedReleaseVersion1.Slug, publicationViewModel.Releases[0].Slug);
-                    Assert.Equal(expectedReleaseVersion1.Title, publicationViewModel.Releases[0].Title);
+                    Assert.Equal(release2022.Versions[1].Id, publicationViewModel.LatestReleaseId);
+
+                    Assert.Equal(release2022.Id, publicationViewModel.Releases[0].Id);
+                    Assert.Equal(release2022.Slug, publicationViewModel.Releases[0].Slug);
+                    Assert.Equal(release2022.Title, publicationViewModel.Releases[0].Title);
+
+                    Assert.Equal(release2020.Id, publicationViewModel.Releases[1].Id);
+                    Assert.Equal(release2020.Slug, publicationViewModel.Releases[1].Slug);
+                    Assert.Equal(release2020.Title, publicationViewModel.Releases[1].Title);
 
-                    Assert.Equal(expectedReleaseVersion2.Id, publicationViewModel.Releases[1].Id);
-                    Assert.Equal(expectedReleaseVersion2.Slug, publicationViewModel.Releases[1].Slug);
-                    Assert.Equal(expectedReleaseVersion2.Title, publicationViewModel.Releases[1].Title);
+                    var releaseSeries = publicationViewModel.ReleaseSeries;
 
-                    Assert.Equal(3, publicationViewModel.ReleaseSeries.Count);
+                    Assert.Equal(3, releaseSeries.Count);
```

Comment:
where did `2021` go? I've probably missed something

Reply:
The data that's setup contains a publication with 3 releases.

2020 has 1 published version
2021 has 0 published versions and 1 draft
2022 has 2 published versions and 1 draft

So slightly above this, `Assert.Equal(2, publicationViewModel.Releases.Count);` and the assertions following it check that `publicationViewModel.Release` contain `2022` and `2020`.

Then here `Assert.Equal(3, releaseSeries.Count)` checks that `publicationViewModel.ReleasesSeries` contains `2022`, `2020`, AND the legacy link which is also set up. See `.WithLegacyLinks([legacyLink])`.

`2021` is missing from both `publicationViewModel.Releases` and `publicationViewModel.ReleaseSeries` because this is a public facing method and it filters out unpublished releases.